src/us/kbase/workspace/test/controllers/handle/HandleServiceController.java

```diff
@@ -35,115 +34,118 @@
 	
 	private final Process handleService;
 	private final int handleServicePort;
-	private final Process handleManager;
-	private final int handleManagerPort;
 	
 	private final Path tempDir;
 	
-	private final static String DB = "hsi";
-	private final static String TABLE = "Handle";
-	private final static String USER = "hsitest";
-	private final static String PWD = "hsi-pass-test";
+	private final static String DB = "handle_db";
```

Comment:
I think this should be a constructor argument to avoid possible name collisions. Allows the user to specify the db name.

Reply:
👍